### PR TITLE
Show incorrect guesses

### PR DESCRIPTION
**Pull request by gtaylor1981**
show incorrect guesses
prevent invalid characters & allow copy/paste
add lint script

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
     "scripts": {
         "test": "echo \"Error: no test specified\" && exit 1",
         "start": "webpack-dev-server --mode development --open --hot --port 3000",
-        "build": "mkdir -p dist && webpack --mode production"
```
Comment (benp44) [inline package.json]: Does dropping the `-p` cause problems on multiple builds?

File: package.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
...
+        "build": "mkdir dist && webpack --mode production",
+        "lint": "npx eslint src --ext .js"
```
Comment (benp44) [inline package.json]: Doesn't work on `.jsx`?

**Review by benp44 — APPROVED** [review]
Looks good!

File: package.json (hunk 1 above)
Comment (gtaylor1981) [reply package.json]: Wasn't win10 compatible. Didn't mean to commit. Will revert

File: package.json (hunk 2 above)
Comment (gtaylor1981) [reply package.json]: Wasn't sure of this. Was mainly a reminder. Will fix

File: package.json (hunk 1 above)
Comment (benp44) [reply package.json]: I should setup some kind of containerised dev env 🤔

**Pull request merged**